core/model/collection.py

```diff
@@ -325,10 +325,11 @@ def default_loan_period(self, library, medium=EditionConstants.BOOK_MEDIUM):
         that someone who borrows a non-open-access item from this
         collection has it for this number of days.
         """
-        return (
+        value = (
             self.default_loan_period_setting(library, medium)
             or self.STANDARD_DEFAULT_LOAN_PERIOD
         )
+        return int(value) if value is not None else None
```

Comment:
Do we need this here? Since we're correctly handing this in the DB, we shouldn't need to force the type here. If we're worried about values that didn't get correctly saved, maybe a migration to clean them up would be better?

Reply:
Ok, there is a logistical problem here. To target all the settings available the best course of action would be to load every settings_dict into their pydantic model and writing the `.dict()` back into the database.
However, the settings models are all defined within the application codebase which would lead to importing the ORM models into the migration code, which is bad practice.
We can either
1. Refactor the models into their own files 
2. Target very specific settings values individually

Reply:
I'm going to use strategy 2, because 1 looks like a refactor we don't need.